tools/common/StringUtils.cs

```diff
@@ -84,7 +84,6 @@ public static string QuoteForProcess (string f)
 			foreach (var c in f) {
 				if (c == '"') {
 					s.Append ('\\');
```

Comment:
In your example, you used single backslash `\` but it looks like here you are using double backslash. Is this desired?